HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -1746,13 +1746,16 @@
     <sch:title>[HeatingDetailedPerformanceDataPointWithCapacity]</sch:title>
     <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACPlant/*/h:HeatingDetailedPerformanceData/h:PerformanceDataPoint/h:Capacity'>
       <sch:assert role='ERROR' test='count(../../../h:HeatingCapacity) = 1'>Expected 1 element(s) for xpath: ../../../HeatingCapacity</sch:assert>
+      <sch:assert role='ERROR' test='number(../../../h:HeatingCapacity) = number(../../h:PerformanceDataPoint[h:OutdoorTemperature=47 and h:CapacityDescription="nominal"]/h:Capacity)'>Expected ../../../HeatingCapacity to be equal to ../../PerformanceDataPoint[OutdoorTemperature=47 and CapacityDescription="nominal"]/Capacity</sch:assert>
+      <sch:assert role='ERROR' test='(number(../../../h:HeatingCapacity17F) = number(../../h:PerformanceDataPoint[h:OutdoorTemperature=17 and h:CapacityDescription="nominal"]/h:Capacity)) or not (../../../h:HeatingCapacity17F)'>Expected ../../../HeatingCapacity17F to be equal to ../../PerformanceDataPoint[OutdoorTemperature=17 and CapacityDescription="nominal"]/Capacity</sch:assert>
```

Comment:
Good thinking. Ideally we would now remove the requirement that `HeatingCapacity` and `CoolingCapacity` is required (and update the docs) since we get the same data from the detailed performance data. No point requiring the same value to be specified two different places. In defaults.rb, we can populate `HeatingCapacity` and `CoolingCapacity` from the detailed performance datapoints. And then all the asserts would be similar to the `HeatingCapacity17F` assert and handle the fact that the input is optional (e.g., ` or not (../../../h:HeatingCapacity17F)`).